src/main/java/frc/robot/RobotContainer.java

```diff
@@ -95,15 +108,48 @@ private void configureBindings() {
         .back()
         .onTrue(Commands.runOnce(() -> s_DriveTrain.resetGyro(), s_DriveTrain));
 
-
-
     m_driverController.x().whileTrue(this.m_autoRoutines.lineUpShooter());
-   // m_driverController.x().whileTrue(new AlignShooter(s_DriveTrain));
+    
+    // m_driverController.x().whileTrue(new AlignShooter(s_DriveTrain));
 
     m_SecondaryController.rightBumper().toggleOnTrue(new Intake(s_IntakeShooter));
+    m_SecondaryController.povDown().onTrue(s_IntakeShooter.limitSwitchOverride(true));
+    m_SecondaryController.povDown().onFalse(s_IntakeShooter.limitSwitchOverride(false));
+    // TODO Remove repeated line below? - Josh
     m_SecondaryController.back().onTrue(s_Climber.zeroMotorsCommand());
     m_SecondaryController.leftBumper().onTrue(s_IntakeShooter.shoot());
     m_SecondaryController.back().onTrue(s_Climber.zeroMotorsCommand());
+
+    m_SecondaryController.povRight().whileTrue(s_IntakeShooter.flushForward());
+    m_SecondaryController.povLeft().whileTrue(s_IntakeShooter.flushBackward());
+
+    m_driverController.start().and(m_driverController.back()).onTrue(swapControllersCommand());
+
+  }
+
+  private Command swapControllersCommand() {
+    // TODO Marcus, I can only solve this issue by adding "extends subsystemBase" to
```

Comment:
The reason for this is that you aren't supposed to put control logic here

Reply:
Ok